nerdlandbot/commands/GuildData.py

```diff
@@ -27,6 +29,8 @@ def __init__(self, guild_id: int):
         self.purgers = dict()
         self.bot_admins = []
         self.culture = "en"
+        self.church_channel = int
```

Comment:
Probably want to set this to `None`, because now you're setting it to the `type` type:
```python
>>> type(int)
# <class 'type'>
```